_schemaV2.graphql

```diff
@@ -10207,6 +10207,8 @@ enum MyCollectionArtworkSorts {
 
 # A connection to a list of items.
 type MyCollectionConnection {
+  artistsCount: Int!
+  artworksCount: Int!
```

Comment:
This is the only relevant change. The rest is only refactoring (moving `myCollectionInfo` into its own file).